cmd/chirpstack-application-server/cmd/configfile.go

```diff
@@ -207,6 +207,20 @@ id="{{ .ApplicationServer.ID }}"
     # The login label is used in the web-interface login form.
     login_label="{{ .ApplicationServer.UserAuthentication.OpenIDConnect.LoginLabel }}"
 
+    # Use UserInfo.
+    #
+    # Use a UserInfo call to retrieve the claim rather than extracting the claim from
+    # the idToken.
+    use_userinfo="{{ .ApplicationServer.UserAuthentication.OpenIDConnect.UseUserInfo }}"
+
+    # Assume email_verified.
+    #
+    # Assume that the email_verified field is always true, regardless of whether it
+    # is included in the token.
+    #
+    # This is required for identity providers that don't supply this claim, such as
+    # Microsoft Azure AD.
+    use_userinfo="{{ .ApplicationServer.UserAuthentication.OpenIDConnect.AssumeEmailVerified }}"
```

Comment:
This should be `assume_email_verified`. As the value is a bool, there should not be `"` around the value. The expected would be `assume_email_verified=true` or `assume_email_verified=false`.

Reply:
Good spot, thanks.